### PR TITLE
Add loop_index

### PR DESCRIPTION
**Pull request by hjopel**
Allows minijanus to specify a loop index as documented in https://github.com/meetecho/janus-gateway/blob/master/conf/janus.jcfg.sample.in#L108

### CODE REVIEW

File: minijanus.js [hunk 1]
```diff
@@ -11,8 +11,8 @@ function JanusPluginHandle(session) {
 }
 
 /** Attaches this handle to the Janus server and sets its ID. **/
-JanusPluginHandle.prototype.attach = function(plugin) {
-  var payload = { plugin: plugin, "force-bundle": true, "force-rtcp-mux": true };
+JanusPluginHandle.prototype.attach = function(plugin, loop_index) {
+  var payload = { plugin: plugin, loop_index: loop_index, "force-bundle": true, "force-rtcp-mux": true };
```
Comment (vincentfretin) [inline minijanus.js:15]: Is this working correctly on janus side if we give `loop_index: undefined`?
It's probably better to not include the key at all in the payload if it's undefined, less bytes to send :)
Do this:
```
if (loop_index) {
  payload.loop_index = loop_index;
}
```

**Review by vincentfretin — CHANGES_REQUESTED** [review]

**Review by vincentfretin — CHANGES_REQUESTED** [review]
At first I thought the key would be in the json, but actually no, when you serialize an object that have a key with undefined value, the key is discarded:
```js
JSON.stringify({"force-rtcp-mux": true, loop_index: undefined})
'{"force-rtcp-mux":true}'
```

so your previous version was good.
The new version with the condition is actually wrong, if loop_index is 0, it won't set it.
Can you please revert back to the previous version and I'll merge, I'm so sorry.

**Comment by hjopel** [comment]
> At first I thought the key would be in the json, but actually no, when you serialize an object that have a key with undefined value, the key is discarded:
> 
> ```js
> JSON.stringify({"force-rtcp-mux": true, loop_index: undefined})
> '{"force-rtcp-mux":true}'
> ```
> 
> so your previous version was good. The new version with the condition is actually wrong, if loop_index is 0, it won't set it. Can you please revert back to the previous version and I'll merge, I'm so sorry.

that's true, I should've also caught that

**Review by vincentfretin — APPROVED** [review]
Thank you.

**Pull request merged**